src/components/InlineCode.tsx

```diff
@@ -45,7 +53,7 @@ const Code = styled.code<{fillLevel: FillLevel}>(({ theme, fillLevel }) => ({
   paddingTop: `${PADDING_EMS}em`,
   paddingBottom: `${PADDING_EMS}em`,
   color: theme.colors['text-light'],
-  borderColor: theme.colors[fillLevelToBorderColor[fillLevel]],
+  borderColor: theme.colors[parentFillLevelToBorderColor[parentFillLevel]],
```

Comment:
It doesn't change in the story, probably because it was not modified.

Reply:
Good catch, and I appreciate the thorough checking. :) PTAL.